vllm/model_executor/models/mlp_speculator.py

```diff
@@ -95,7 +101,10 @@ def __init__(self, config: MLPSpeculatorConfig, **kwargs) -> None:
             self.proj = nn.ModuleList([proj_first] + [proj_tied] *
                                       (self.max_speculative_tokens - 1))
 
-            head = ParallelLMHead(self.vocab_size, self.inner_dim, bias=False)
+            head = ReplicatedLinear(self.inner_dim,
```

Comment:
Can we reuse `quant_config` in [ParallelLMHead](https://github.com/vllm-project/vllm/blob/9ba0817ff1eb514f51cc6de9cb8e16c98d6ee44f/vllm/model_executor/layers/vocab_parallel_embedding.py#L441C18-L441C30) or is there some limitation?

Reply:
`ParallelLMHead` is a `VocabParallelEmbedding`, for it to support quantization, a corresponding quantized embedding is needed but it does not exist (and it is not used by `MLPSpeculator`)